src/screens/DomainScreen/elements/Header/elements/ActionButtonGroup.js

```diff
@@ -115,7 +115,7 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center',
     borderWidth: 1,
-    borderColor: COLORS.anon_primary,
+    borderColor: COLORS.signed_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the change in border color from `COLORS.anon_primary` to `COLORS.signed_primary` should be fine if it aligns with your new color scheme.

```diff
-    borderColor: COLORS.anon_primary,
+    borderColor: COLORS.signed_primary,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->